### PR TITLE
Configuration Rspect and test book

### PR DESCRIPTION
**Pull request by AleFalcon**
## Summary
* Configuration Rspect
* Creation test book by validation of attributes
## Trello Card
https://trello.com/c/LU7VMECS/20-crear-tests-para-el-modelo-de-book

### CODE REVIEW

File: spec/factories/book.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+FactoryBot.define do
+  factory :book do
+    gender { Faker::Book.genre }
+    author { Faker::Book.author }
+    image { Faker::String.random }
+    title { Faker::Book.title }
+    editor { Faker::Book.publisher }
+    year { (Time.zone.today - Faker::Number.number(3).to_i.days).to_s }
```
Comment (nicozar95) [inline spec/factories/book.rb]:
```suggestion
    year { Faker::Date.forward(23) }
```

File: spec/factories/user.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+FactoryBot.define do
+  factory :user do
+    email { Faker::Internet.email }
+    password { '123456789' }
```
Comment (nicozar95) [inline spec/factories/user.rb]: Use faker for this
```suggestion
    password { Faker::Internet.passwor }
```

Comment (AleFalcon) [reply spec/factories/user.rb]: Ready

File: spec/factories/book.rb (hunk 1 above)
Comment (AleFalcon) [reply spec/factories/book.rb]: Ready

**Pull request merged**